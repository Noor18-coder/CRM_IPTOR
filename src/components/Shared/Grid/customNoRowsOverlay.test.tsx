import renderer from 'react-test-renderer';
import CustomNoRowsOverlay from './customNoRowsOverlay';

const noRowsMessageFunc = jest.fn();

describe('[Shared] customNoRowsOverlay', () => {
  it('should renders correctly', () => {
    const tree = renderer.create(<CustomNoRowsOverlay noRowsMessageFunc={noRowsMessageFunc} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
