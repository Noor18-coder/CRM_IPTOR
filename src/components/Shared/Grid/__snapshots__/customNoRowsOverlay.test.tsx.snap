// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[Shared] customNoRowsOverlay should renders correctly 1`] = `
<div>
  <i
    className="far fa-frown"
  >
     
  </i>
</div>
`;
